File: Question3.c

#include <stdio.h>
#include <stdbool.h>

bool CheckEqual(int iNo1, int iNo2)
{
    if (iNo1 == iNo2)
    {
        return true;
    }
    else
    {
        return false;
    }
}   

int main()
{
    int iValue1= 0 ,iValue2 = 0 ;
    bool bRet = false;

    printf("Enter Numbers : ");
    scanf("%d %d",&iValue1 , &iValue2);

    bRet = CheckEqual(iValue1, iValue2);
    if (bRet == true)
    {
        printf ("Equal");
    }
    else
    {
        printf("Not Equal");
    }


   
    return 0 ;
}

File: Assignment 17/Question3.java
import java.util.Scanner;

class Logic
{
    
    public int CheckMax(int iNo1 ,int  iNo2)
    {
        if (iNo1>iNo2)
        {
            return iNo1 ;
        }
        else
        {
            return iNo2;
        }
        
    
        
       
    }
}

class Question3 
{
    public static void main(String[] args) 
    {
        
        Scanner sobj = new Scanner(System.in);
        int iValue1 = 0 ;
        int iValue2 = 0 ;
        int iRet = 0 ;
     

        System.out.println("Enter your Number :");
        iValue1 = sobj.nextInt();

        System.out.println("Enter your Number :");
        iValue2 = sobj.nextInt();

        Logic lobj = new Logic();

        iRet = lobj.CheckMax(iValue1,iValue2);

       
        
          System.out.println("The maximum no is : " + iRet);  
        
    }    
}
